### DISCUSSION TITLE
Trouble Logging in

### DISCUSSION THREAD

**Discussion by chlanman** [widewing/ha-toyota-na#56] [Q&A]
Does this work with Canada's Toyota Connected App?
Its not allowing me to login?

Thanks

**Comment by android52** [comment]
Did anyone ever answer this?  

I'm in Canada with the Connected App, and can't seem to create a US account with my VIN.  I'm also suspecting we need a US account to use the Integration, but would love to have confirmation.

**Reply by jimmycunny** [reply]
I'm also in the same boat.  I'm in Canada and can not login.

**Reply by android52** [reply]
Thanks.  Did you buy the car new or used?  What model/year?  I'm in a 22 Highlander so the "older" connected system I believe and since it was used the free remote connect trial had run out so I'm not sure if it would work with that.  

Any chance you have remote connect enabled and confirm it's not just requiring a subscription?

**Reply by vager88** [reply]
So I'm currently stuck not able to log in, I just get an error.  I'm in Canada with a 2023 model, so I currently have access to the full connected services. So not being able to log in from Canada may not related to the trial being active or not.   Now initially I was connecting via my Google account, but I just did the forgot password so that I could set one up.  I still can't connect.

**Comment by android52** [comment]
Did you review Issue #52 ?  I followed the instructions in that discussion thread and was able to get past the "Not Logged In" (about 2 months ago so who knows if things have changed).

I was stuck after that (and still am) with messages about waiting for vehicle information (or along those lines) as I put in my response over there.

Hopefully that thread can help you and with an active subscription to Remote Connect (another thread said that is required for the integration) maybe you can test and confirm if this works in Canada for you, my Remote Connect subscription was expired when I bought the car (used) so I can't confirm that is the problem.
